examples/unix/c11/z_ping.c

```diff
@@ -79,8 +100,8 @@ int main(int argc, char** argv) {
     _z_mutex_unlock(&mutex);
     z_free(results);
     z_free(data);
-    z_drop(z_move(sub));
-    z_drop(z_move(pub));
```

Comment:
z_drop should be favored in c11 examples

Reply:
Done!